CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
-cmake_minimum_required(VERSION 3.0.2)                        
-cmake_policy(SET CMP0048 NEW)
+cmake_minimum_required(VERSION 3.20)                        
+cmake_policy(SET CMP0074 NEW)
 
-project(xcompact3d LANGUAGES Fortran)
+project(Incompact3d LANGUAGES Fortran)
 
-set(AUTHOR "Stefano Rolfo;Charles Moulinec")
-set(AUTHOR_DETAILS "stefano.rolfo@stfc.ac.uk;charles.moulinec@stfc.ac.uk")
-set(DESCRIPTION "Building xcompact3d using cmake")
+set(AUTHOR "Stefano Rolfo;Charles Moulinec;Paul Bartholomew")
+set(AUTHOR_DETAILS "stefano.rolfo@stfc.ac.uk;charles.moulinec@stfc.ac.uk;p.bartholomew@epcc.ed.ac.uk")
+set(DESCRIPTION "Building Incompact3d using cmake")
```

Comment:
We should update this to atleast the full list of core developers or delete. It's not fair to only acknowledge a few, I think we have a contributor list somewhere

Reply:
https://github.com/xcompact3d/Incompact3d/blob/master/CONTRIBUTORS

can we remove this AUTHOR thing then?

Reply:
I would remove the authors for the CMake and keep the file.